java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+- fix action chains for saltssh minions (bsc#a1182200)
```

Comment:
There seems to be a typo in the bugnumber. The `a` is maybe just too much?

Reply:
Seem to be fixed now. I will merge as soon as tests are passing.